repository_service_tuf_api/api/metadata.py

```diff
@@ -28,7 +29,7 @@
 )
 def post(
     payload: metadata.MetadataPostPayload,
-    _user=Security(auth, scopes=[SCOPES_NAMES.write_metadata.value]),
+    _user=Security(auth, scopes=[SCOPES_NAMES.write_metadata_sign.value]),
```

Comment:
This endpoint is for the metadata update.
```suggestion
    _user=Security(auth, scopes=[SCOPES_NAMES.write_metadata.value]),
```